lib/core/modules/coderunner/codeRunner.js

```diff
@@ -41,6 +41,10 @@ class CodeRunner {
       if (!cb) {
         cb = function() {};
       }
+      if (code.startsWith('await')) {
+        code = code.substring(5); // remove await keyword
+        code += '.then(console.log).catch(console.error)'; // Add promise catch
```

Comment:
is it possible to assign the result if doing it that way
``` let accounts = await web3.eth.getAccounts()```

Reply:
Hum, didn't think of that. It is not possible for now. Though, not sure if anyone would do that. I'll try to see if it's possible to use variables right now in the current console